### PR TITLE
deleted .gitignore, added test_particles.py

### PR DESCRIPTION
**Pull request by jRicciL**

### CODE REVIEW

File: test_particles.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+def test_from_file():
+    particles =  Particles.from_file(file_name='sample_config1.xyz')
+    # File includes 800 particles
+    expected_num_particles = 800
+    calculated_num_particles = particles.num_particles
+    assert (expected_num_particles == calculated_num_particles)
+
+def test_from_random():
```
Comment (b-butler) [inline test_particles.py:15]: We should check that particles are within the box dimensions.

**Review by b-butler — CHANGES_REQUESTED** [review]

File: test_particles.py (hunk 1 above)
Comment (jRicciL) [reply test_particles.py:15]: Sure, you're right, thank you Brandon...

File: test_particles.py [hunk 2]
```diff
@@ -5,17 +5,23 @@
...
     expected_num_particles = 3
     calculated_num_particles = particles.num_particles
-    assert (expected_num_particles == calculated_num_particles)
+    
+    # Testing coordinates are inside the box dimensions
+    expected_value = True
+    calculated_value = (np.abs(particles.coordinates) <= box_dims).all()
+
+    assert(expected_num_particles == calculated_num_particles)
+    assert(calculated_value == expected_value)
```
Comment (b-butler) [inline test_particles.py]: You could just assert `calculated_value` since it is a boolean.

**Review by b-butler — APPROVED** [review]
Looks awesome.

**Pull request merged**